src/components/button-toggle/button-toggle.pw.tsx

```diff
@@ -0,0 +1,858 @@
+      .filter({ hasText: "button after" });
+
+    await buttonAfter.focus();
+    await page.keyboard.press("Shift+Tab");
+    await expect(buttonToggleButton(page).nth(0)).toBeFocused();
+    await page.keyboard.press("Shift+Tab");
+    await expect(buttonBefore).toBeFocused();
+  });
+
+  test(`should only allow the selected button to be tabbed to when it is selected`, async ({
+    mount,
+    page,
+  }) => {
+    await mount(<WithOutsideButtons />);
+
+    const buttonBefore = page
+      .getByRole("button")
+      .filter({ hasText: "button before" });
+    const buttonAfter = page
+      .getByRole("button")
+      .filter({ hasText: "button after" });
+
+    await buttonTogglePreview(page).nth(1).click();
+    await buttonBefore.focus();
+    await page.keyboard.press("Tab");
+    await expect(buttonToggleButton(page).nth(1)).toBeFocused();
+    await page.keyboard.press("Tab");
+    await expect(buttonAfter).toBeFocused();
+  });
+
+  test(`should only allow the selected button to be shift-tabbed to when one is selected`, async ({
+    mount,
+    page,
+  }) => {
+    await mount(<WithOutsideButtons />);
+
+    const buttonBefore = page
+      .getByRole("button")
+      .filter({ hasText: "button before" });
+    const buttonAfter = page
+      .getByRole("button")
+      .filter({ hasText: "button after" });
+
+    await buttonTogglePreview(page).nth(1).click();
+    await buttonAfter.focus();
+    await page.keyboard.press("Shift+Tab");
+    await expect(buttonToggleButton(page).nth(1)).toBeFocused();
+    await page.keyboard.press("Shift+Tab");
+    await expect(buttonBefore).toBeFocused();
+  });
+
+  test(`should cycle through the buttons in the group when using the right arrow key`, async ({
+    mount,
+    page,
+  }) => {
+    await mount(<ButtonToggleGroupComponent />);
+
+    await buttonToggleButton(page).nth(0).click();
+    await page.keyboard.press("ArrowRight");
+    await expect(buttonToggleButton(page).nth(1)).toBeFocused();
+    await page.keyboard.press("ArrowRight");
+    await expect(buttonToggleButton(page).nth(2)).toBeFocused();
+    await page.keyboard.press("ArrowRight");
+    await expect(buttonToggleButton(page).nth(0)).toBeFocused();
+  });
+
+  test(`should cycle through the buttons in the group, selecting each one, when using the left arrow key`, async ({
+    mount,
+    page,
+  }) => {
+    await mount(<ButtonToggleGroupComponent />);
+
+    await buttonToggleButton(page).nth(0).click();
+    await page.keyboard.press("ArrowLeft");
+    await expect(buttonToggleButton(page).nth(2)).toBeFocused();
+    await page.keyboard.press("ArrowLeft");
+    await expect(buttonToggleButton(page).nth(1)).toBeFocused();
+    await page.keyboard.press("ArrowLeft");
+    await expect(buttonToggleButton(page).nth(0)).toBeFocused();
+  });
+});
+
+test.describe("Accessibility tests for group component", () => {
+  test(`should pass accessibility tests for Button-Toggle-Group default story`, async ({
```

Comment:
Same changes as requested for the previous accessibility tests. 

```suggestion
  test("should pass accessibility tests for default example", async ({
```